e2e/adversarial_test.go

```diff
@@ -64,13 +64,13 @@ func (suite *AdversarialTestSuite) SetupTest() {
 	suite.chainC = suite.coordinator.GetChain(wasmibctesting.GetChainID(2))
 
 	storeCodes := func(chain *wasmibctesting.TestChain, bridge *sdk.AccAddress) {
-		resp := chain.StoreCodeFile("../artifacts/cw_ics721_bridge.wasm")
+		resp := chain.StoreCodeFile("../artifacts/ics721_base.wasm")
```

Comment:
TODO: e2e tests crashed with this error: 

```sh
Received unexpected error:
github.com/cosmos/cosmos-sdk/baseapp.newOutOfGasRecoveryMiddleware.func1
...
out of gas in location: Compiling wasm bytecode; gasWanted: 3000000, gasUsed: 3110541: out of gas
```

Error only pops up on storing ics721 contract - storing cw721 is fine. Error occured after adding additional logic for adding ClassData.

Found this hint: https://github.com/cosmos/cosmos-sdk/issues/16020

Reply:
will take a look on this

Reply:
> will take a look on this

Steps to reproduce error:
```
cd e2e
./test.sh
```